src/wp-admin/user-edit.php

```diff
@@ -241,213 +255,225 @@
 		<td>
-			<label for="syntax_highlighting"><input name="syntax_highlighting" type="checkbox" id="syntax_highlighting" value="false" <?php if ( ! empty( $profileuser->syntax_highlighting ) ) checked( 'false', $profileuser->syntax_highlighting ); ?> /> <?php _e( 'Disable syntax highlighting when editing code' ); ?></label>
+			<label for="syntax_highlighting"><input name="syntax_highlighting" type="checkbox" id="syntax_highlighting" value="false" <?php checked( 'false', $profileuser->syntax_highlighting ); ?> /> <?php _e( 'Disable syntax highlighting when editing code' ); ?></label>
 		</td>
 	</tr>
 <?php endif; ?>
-<?php if ( count($_wp_admin_css_colors) > 1 && has_action('admin_color_scheme_picker') ) : ?>
+		<?php if ( count( $_wp_admin_css_colors ) > 1 && has_action( 'admin_color_scheme_picker' ) ) : ?>
 <tr class="user-admin-color-wrap">
-<th scope="row"><?php _e('Admin Color Scheme')?></th>
-<td><?php
-	/**
-	 * Fires in the 'Admin Color Scheme' section of the user editing screen.
-	 *
-	 * The section is only enabled if a callback is hooked to the action,
-	 * and if there is more than one defined color scheme for the admin.
-	 *
-	 * @since WP-3.0.0
-	 * @since WP-3.8.1 Added `$user_id` parameter.
-	 *
-	 * @param int $user_id The user ID.
-	 */
-	do_action( 'admin_color_scheme_picker', $user_id );
-?></td>
+<th scope="row"><?php _e( 'Admin Color Scheme' ); ?></th>
+<td>
+			<?php
+			/**
+			 * Fires in the 'Admin Color Scheme' section of the user editing screen.
+			 *
+			 * The section is only enabled if a callback is hooked to the action,
+			 * and if there is more than one defined color scheme for the admin.
+			 *
+			 * @since WP-3.0.0
+			 * @since WP-3.8.1 Added `$user_id` parameter.
+			 *
+			 * @param int $user_id The user ID.
+			 */
+			do_action( 'admin_color_scheme_picker', $user_id );
+			?>
+</td>
 </tr>
-<?php
-endif; // $_wp_admin_css_colors
-if ( !( IS_PROFILE_PAGE && !$user_can_edit ) ) : ?>
+			<?php
+	endif; // $_wp_admin_css_colors
+		if ( ! ( IS_PROFILE_PAGE && ! $user_can_edit ) ) :
+			?>
 <tr class="user-comment-shortcuts-wrap">
 <th scope="row"><?php _e( 'Keyboard Shortcuts' ); ?></th>
-<td><label for="comment_shortcuts"><input type="checkbox" name="comment_shortcuts" id="comment_shortcuts" value="true" <?php if ( ! empty( $profileuser->comment_shortcuts ) ) checked( 'true', $profileuser->comment_shortcuts ); ?> /> <?php _e('Enable keyboard shortcuts for comment moderation.'); ?></label> <?php _e('<a href="https://codex.wordpress.org/Keyboard_Shortcuts" target="_blank">More information</a>'); ?></td>
+<td><label for="comment_shortcuts"><input type="checkbox" name="comment_shortcuts" id="comment_shortcuts" value="true" <?php checked( 'true', $profileuser->comment_shortcuts ); ?> /> <?php _e( 'Enable keyboard shortcuts for comment moderation.' ); ?></label> <?php _e( '<a href="https://codex.wordpress.org/Keyboard_Shortcuts" target="_blank">More information</a>' ); ?></td>
```

Comment:
The default value is set to true. No need to check with `if`.  `checked()` has defaults to this effect.

https://developer.wordpress.org/reference/functions/checked/

```
function checked( $checked, $current = true, $echo = true ) {
    return __checked_selected_helper( $checked, $current, $echo, 'checked' );
}
```

Reply:
A value of "true" is not the same as `true` as the default parameter of `checked`.

Reply:
The data types are definitely different. However, the `if` check doesn't check for the value being set and data type being same; It goes to find if the value has been set and that will always be set as "true".

Reply:
The data types are definitely different. However, the if check doesn't check for the value being set and data type being same; It goes to find if the value has been set and that will always be set as "true".

Reply:
I see that since it's checking the value in the database, not the value from the form, and the database contains a string instead of a boolean, then the `empty` check does nothing.

Reply:
Resolving this since the previous behavior of the code is still intact.